content/how-to/dedicated-hosting/delete-vpn-user-myrackspace-portal/index.md

```diff
@@ -38,24 +45,28 @@ This article describes how to delete a VPN user by using a ticket template.
    b. If you need to delete the VPN user from all firewalls on the account,
    ensure that you select all firewalls in this field.
 
+   <img width="410" alt="selectfirewall" src="/support/how-to/delete-vpn-user-myrackspace-portal/deletevpn6.png">
+
 7. Enter your desired VPN username in the **VPN Username** field.
 
    a. The VPN username is case-sensitive and requires at least three characters.
 
    b. Do not include a question mark (?) or space ( ) within the VPN username field.
 
-8. Click **Create Ticket** at the bottom of the page.
+   <img width="423" alt="usernamecredential" src="/support/how-to/delete-vpn-user-myrackspace-portal/deletevpn7.png">
+
+
+8. Click **Submit** at the bottom of the page.
 
    After the ticket generates, you see your ticket list in
    the MyRackspace Portal. Automation runs in the background while the process
    creates the ticket, which typically takes less than a minute to complete.
    After you refresh your browser, the ticket *Delete VPN User on Firewall Request (from template)*
-   is in **Confirm Solved** status.
+   is in **Solved** status.
```

Comment:
So it went to `Solved` status first? If so, I think that is correct. Customer needs to close it and then it goes to Closed I think. After several days of being open I think it will autoclose.

Reply:
Yes, checked.